.eslintrc.js

```diff
@@ -111,7 +112,6 @@ module.exports = {
     "prefer-rest-params": "error",
     "prefer-spread": "error",
     "prefer-template": "error",
-    "sort-requires/sort-requires": "error",
-    strict: ["error", "global"],
+    "sort-imports": ["error", { allowSeparatedGroups: true }],
```

Comment:
[note] I've enabled `allowSeparatedGroups` because the pattern grouping imports by an empty line are common in our code base.

https://eslint.org/docs/rules/sort-imports#allowseparatedgroups